<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogClass</class>
 <widget class="QDialog" name="DialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Oscilloscope</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="MyGraph" name="Display" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="palette">
          <palette>
           <active>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>10</red>
               <green>50</green>
               <blue>10</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>10</red>
               <green>50</green>
               <blue>10</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Base">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>10</red>
               <green>50</green>
               <blue>10</blue>
              </color>
             </brush>
            </colorrole>
            <colorrole role="Window">
             <brush brushstyle="SolidPattern">
              <color alpha="255">
               <red>10</red>
               <green>50</green>
               <blue>10</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QCheckBox" name="Ch1">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>Ch1</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QDial" name="Ch1Offset">
              <property name="minimum">
               <number>-50</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="notchTarget">
               <double>3.700000000000000</double>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLCDNumber" name="Ch1Vdiv">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QDial" name="Ch2Offset">
              <property name="minimum">
               <number>-50</number>
              </property>
              <property name="maximum">
               <number>50</number>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="2">
             <widget class="QLCDNumber" name="Ch2Vdiv">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QCheckBox" name="Ch2">
              <property name="text">
               <string>Ch2</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QCheckBox" name="Ch3">
              <property name="text">
               <string>Ch3</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QCheckBox" name="Ch4">
              <property name="text">
               <string>Ch4</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <widget class="QDial" name="Ch3Offset">
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QDial" name="Ch4Offset">
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QLCDNumber" name="Ch3Vdiv">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="4">
             <widget class="QLCDNumber" name="Ch4Vdiv">
              <property name="segmentStyle">
               <enum>QLCDNumber::Flat</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QDial" name="Ch1Gane">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QDial" name="Ch2Gane">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QDial" name="Ch3Gane">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QDial" name="Ch4Gane">
              <property name="maximum">
               <number>10</number>
              </property>
              <property name="notchesVisible">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Gane</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>YOffset</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>mV/div</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QCheckBox" name="Dif">
              <property name="text">
               <string>Dif</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::WinPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0" colspan="2">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>AdcSpeed</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>XScale</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>XOffset</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QDial" name="AdcSpeed">
                <property name="maximum">
                 <number>6</number>
                </property>
                <property name="pageStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>4</number>
                </property>
                <property name="notchesVisible">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QDial" name="XScale">
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="pageStep">
                 <number>9</number>
                </property>
                <property name="value">
                 <number>1</number>
                </property>
                <property name="notchesVisible">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QDial" name="XOffset">
                <property name="minimum">
                 <number>-500</number>
                </property>
                <property name="maximum">
                 <number>500</number>
                </property>
                <property name="notchTarget">
                 <double>50.000000000000000</double>
                </property>
                <property name="notchesVisible">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="Start">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="SavePic">
                <property name="text">
                 <string>Save Pic</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>SampleRate, Ksps</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="SampleRate">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="mode">
                 <enum>QLCDNumber::Dec</enum>
                </property>
                <property name="segmentStyle">
                 <enum>QLCDNumber::Flat</enum>
                </property>
                <property name="value" stdset="0">
                 <double>0.000000000000000</double>
                </property>
                <property name="intValue" stdset="0">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="VrefLabel">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Vref, V</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="VrefList">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string>2.56</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>5</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraph</class>
   <extends>QWidget</extends>
   <header>graph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
